lib/secureWebtask.js

```diff
@@ -0,0 +1,54 @@
+const _ = require('lodash');
+const Bluebird = require('bluebird');
+const Request = Bluebird.promisifyAll(require('request'));
+const Cli = require('structured-cli');
+
+const MIDDLWARE_COMPILER = '@webtask/middleware-compiler';
+const MIDDLWARE_COMPILER_VERSION = '^1.2.1';
+const JWT_MIDDLEWARE = '@webtask/jwt-middleware';
+const JWT_MIDDLEWARE_VERSION = '^1.4.0';
+
+module.exports = secureWebtask = (args, options) => {
+    return Request.getAsync(args.profile.url + '/api/description').then(res => { 
+        addMeta(args, res, options); 
+    }).catch(err => {
+        throw new Cli.error.serverError(`Unable to secure webtask: ${err.message}`);
+    });
+}
+
+function addMeta(args, res, options) {
```

Comment:
I can't remember, but is there any situation in which `args.meta` comes in as `undefined`?